conf/nginx.conf

```diff
@@ -20,6 +20,8 @@ location ^~ #LOCATION# {
   add_header X-Robots-Tag none;
   add_header X-Download-Options noopen;
   add_header X-Permitted-Cross-Domain-Policies none;
+  # Add data: to allow /ynhpanel.css to be load due to image on data:base64
+  more_set_headers Content-Security-Policy "default-src data:;";
```

Comment:
wrong syntax, so this CSP (`default-src data:;`) is never applied.

But due to `more_set_headers` syntax, this line completely remove the current CSP header. See https://github.com/openresty/headers-more-nginx-module#more_set_headers

A correct syntax would be:
```
more_set_headers "Content-Security-Policy: default-src data:;";
``` 

but having `default-src data:;` as CSP broke everything !!  To see what current CSP is applied, use `curl -I https://domain.com |grep '^Content-Security-Policy:'`